pkg/api/handlers/app_handler.go

```diff
@@ -33,6 +33,7 @@ func (ah *AppHandler) HandleCreate() rest.Handler {
 	logger.Info("handling dApp create request")
 	handler := func(w http.ResponseWriter, r *http.Request) {
 		data := models.AppDI{}
+		scope := r.Header.Get("")
```

Comment:
Shouldn't this have a key?
```suggestion
		scope := r.Header.Get(rest.HeaderScopeKey)
```

Reply:
true